.github/workflows/conformance.yml

```diff
@@ -19,7 +19,6 @@ on:
 
 env:
   GO_VERSION: "1.18"
-  AWS_REGION: "us-west-2"
```

Comment:
We're just using the default region for all AWS stuff rightnow, so I dropped this. If we ever need to use a non-default region, easy enough to put it back.